src/components/Garden/Stake/index.d.ts

```diff
@@ -0,0 +1,64 @@
+type StakeMovement = {
+  actionId: string
+  actionState: string
+  agreementId: string
+  amount: string
+  collateralState: string
+  createdAt: string | number
```

Comment:
Almost sure it should be just number, i dont found references that being string
```suggestion
  createdAt: number
```

Reply:
<img width="800" alt="Captura de ecrã 2022-04-05, às 11 22 49" src="https://user-images.githubusercontent.com/13184907/161733700-b3fd55e7-b4bd-4ae9-ba63-8ee24976502b.png">

It was a fail safety, its a string.

Reply:
Humm, i see. but it not could lead to people put some date? Like "05/04/2022" on that? Keeping it numbers could avoid that.

Reply:
This comes from the subgraph I think.